### PR TITLE
chore(dependencies): Spring Boot 2.2.4 upgrade

### PR DESCRIPTION
**Pull request by pdelagrave**

### CODE REVIEW

File: igor-web/src/main/java/com/netflix/spinnaker/igor/gcb/GoogleCloudBuildParser.java [hunk 1]
```diff
@@ -48,6 +52,24 @@
     return parse(inputString, destinationClass);
   }
 
+  @SuppressWarnings("unchecked")
+  public final Map serializeBuild(Build inputBuild) {
+    // com.google.api.services.cloudbuild.v1.model.StorageSource.generation is of type Long
```
Comment (cfieber) [inline igor-web/src/main/java/com/netflix/spinnaker/igor/gcb/GoogleCloudBuildParser.java]: @plumpy think you could find the right team to let them know about this?

**Comment by Vity01** [comment]
Just note there is already Spring Boot 2.2.4...

**Comment by pdelagrave** [comment]
> Just note there is already Spring Boot 2.2.4...

As yesterday we'll be using 2.2.4: https://github.com/spinnaker/kork/pull/482
The PR titles just haven't been updated.

**Comment by pdelagrave** [comment]
flaky test but otherwise ready

File: igor-web/src/main/java/com/netflix/spinnaker/igor/gcb/GoogleCloudBuildParser.java (hunk 1 above)
Comment (pdelagrave) [reply igor-web/src/main/java/com/netflix/spinnaker/igor/gcb/GoogleCloudBuildParser.java]: Still an issue but moved the fix in test code as this only impacts our tests so far.
https://github.com/spinnaker/igor/blob/c467ba2ad813c3b4abf4e0ab965ab020edbdeb86/igor-web/src/test/groovy/com/netflix/spinnaker/igor/gcb/GoogleCloudBuildTestSerializationHelper.groovy

**Pull request merged**